libkbfs/key_bundle_cache_test.go

```diff
@@ -37,13 +37,25 @@ func TestKeyBundleCacheBasic(t *testing.T) {
 
 	cache := NewKeyBundleCacheStandard(4)
 
-	if checkWkb := cache.GetTLFWriterKeyBundle(tlf, wkbID); checkWkb != nil {
+	checkWkb, err := cache.GetTLFWriterKeyBundle(tlf, wkbID)
+	if err != nil {
```

Comment:
All these checks would be much cleaner using github.com/stretchr/testify/require.go (`require.NoError(t, err)`, `require.NotNil(t, checkWkb)`)